drivers/SmartThings/matter-window-covering/fingerprints.yml

```diff
@@ -5,7 +5,12 @@ matterManufacturer:
     vendorId: 0x130A
     productId: 0x55
     deviceProfileName: window-covering-profile
-
+#Zemismart
+  - id: "Zemismart MT01 Slide Curtain"
+    deviceLabel: Zemismart MT01 Slide Curtain
+    vendorId: 0x139C
+    productId: 0xFFFE
+    deviceProfileName: window-covering-battery
```

Comment:
I believe this should just be `window-covering` (no battery) as per discussion in the latest two comments here: https://smartthings.atlassian.net/browse/MTR-605?focusedCommentId=1416175

Reply:
Updated, thanks @ctowns